### ISSUE TITLE
[CRASH] "Feature Placement" Crash - "No Value Present" - "ctov:medium/village_mountain"

### ISSUE THREAD

**Issue by JacobColahan** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#166]
**Describe the bug**
Running a heavily modded 1.20.1 server. Upgraded CTOV from 3.4.3 to 3.4.4.
Suddenly, we are getting repeated crashes with the same message every single time.

**To Reproduce**
Steps to reproduce the behaviour:
This started when I had a player out traveling and generating chunks.
The server crashed. It would then continue crashing each time that player logged in.
With the player offline, server works fine, until someone else travels to that players previous location. The details in the crash report and behavior while in game suggests that CTOV is the cause of the crash while generating a specific feature.

**Expected behaviour**
CTOV should be generating structures normally, not causing crashes.

**Screenshots**
N/A

**Minecraft Version**
1.20.1

**CTOV Version**
3.4.4

**Add-ons used (Before 1.20)**
N/A

**Do you install Lithostitched?(1.20+)**
Yes

**Log**
[crash-2024-06-26_13.43.41-server.txt](https://github.com/user-attachments/files/15994220/crash-2024-06-26_13.43.41-server.txt)

**Additional context**
Not really sure what else to add. I'll be happy to answer questions though.

**Comment by ChoiceTheorem** [comment]
Can you try resetting that chunk first?

**Comment by JacobColahan** [comment]
Hello! 

Yes I did. 
In the crash log, it gives the coordinates of the player and the coordinates of the chunk that failed to generate.
I didn't quite know what I was doing but played it safe.

I took both sets of coordinates and put them on this site: https://minecraft.tools/en/coordinate-calculator.php

Then, while the server was off, I entered the region, entities, and POI folders within the world directory and deleted the chunks accordingly, thinking that it would reset the chunk.

It did reset the chunk, but it didn't stop the crash. 

The only thing that has allowed us to keep playing was removing the mod entirely.
I'm hoping to add it back once I find a fix.
